src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Argument.java

```diff
@@ -0,0 +1,28 @@
+package io.jenkins.plugins.report.jtreg.main.comparator;
+
+import java.util.Optional;
+
+public class Argument {
+    private final String argumentName;
+    private final String helpMessage;
+
+    private final String argumentFormat;
+
+    public Argument(String argumentName, String helpMessage, Optional<String> argumentFormat) {
```

Comment:
Do I read correclty  you dropped the short like form? LIke -h? Except the -h/--help I would agree on it.